minecraft-common/src/main/java/com/sigmundgranaas/forgero/minecraft/common/utils/DebugUtil.java

```diff
@@ -0,0 +1,15 @@
+package com.sigmundgranaas.forgero.minecraft.common.utils;
+
+import static org.lwjgl.glfw.GLFW.GLFW_CURSOR;
+import static org.lwjgl.glfw.GLFW.GLFW_CURSOR_NORMAL;
+
+import org.lwjgl.glfw.GLFW;
+
+import net.minecraft.client.MinecraftClient;
+
+public class DebugUtil {
+	public static boolean releaseControl() {
+		GLFW.glfwSetInputMode(MinecraftClient.getInstance().getWindow().getHandle(), GLFW_CURSOR, GLFW_CURSOR_NORMAL);
+		return true;
+	}
+}
```

Comment:
What does this class do? Hard to see from the code since I haven't used `GLFW` that much before while modding. I suppose it releases your mouse cursor or something like that?
Would be neat to have JavaDocs here as well.

Reply:
Ah, right it's a silly debugging hack that's needed on linux when placing breakpoints to prevent the game from locking your mouse inside the game